tests/multiparser_html_ts/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,38 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`html-with-ts-script.html 1`] = `
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <script lang="ts">
+    type X = { [
+        K in keyof Y
+    ]: Partial < K >  } ;
+
+    class   Foo< T >{
+
+        constructor ( private foo: keyof Apple ){
+
+
+        }
+    }
+    </script>
+</head>
+<body></body>
+</html>
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script lang="ts">
+      type X = { [K in keyof Y]: Partial<K> };
+      class Foo<T> {
+        constructor(private foo: keyof Apple) {}
+      }
+
```

Comment:
I'm aware of the trailing `hardline` but I don't know a way to remove it without hacking it out of the doc.